workflow-job

```diff
@@ -215,14 +228,20 @@ def isAlive(dockerContainer) {
    }
 }
 
+def provisionDataForIntegrationTest(dockerContainer, jamesCliCmd) {
+    sh "docker exec ${dockerContainer} ${jamesCliCmd} adddomain domain"
+    sh "docker exec ${dockerContainer} ${jamesCliCmd} adduser imapuser@domain password"
+}
+
 flows["${sha1}"] = {
     setDockerHost(dockerIp)
     def buildId = "${branch.toLowerCase()}-${UUID.randomUUID()}"
     def mergeBranch = "MERGE-${buildId}"
     def images = new Images(buildId);
     def containers = new Containers(buildId);
     def jamesLinks = "--link ${containers.cassandra}:cassandra --link ${containers.elasticSearch}:elasticsearch"
-    def findImapPort = "export JAMES_PORT=`docker inspect -f '{{(index (index .NetworkSettings.Ports \"143/tcp\") 0).HostPort}}' ${containers.james}`"
+    def findImapPortCassandra = "export JAMES_PORT=`docker inspect -f '{{(index (index .NetworkSettings.Ports \"143/tcp\") 0).HostPort}}' ${containers.jamesCassandra}`"
```

Comment:
Yes, I will be careful when edit anything. I need to understand before doing anything. Thanks